pkgs/by-name/ev/evil-winrm/package.nix

```diff
@@ -38,6 +57,11 @@ stdenv.mkDerivation rec {
     cp evil-winrm.rb $out/bin/evil-winrm
   '';
 
+  postFixup = lib.optionalString sslLegacyProvider ''
+    wrapProgram $out/bin/evil-winrm --prefix OPENSSL_CONF : "${openssl_conf}"
```

Comment:
I don't think you can change line without `\` at the end, though? Have you tested building with `sslLegacyProvider = true`?